core/schemas/observables/registry_key.py

```diff
@@ -0,0 +1,9 @@
+from core.schemas.observable import Observable
+from core.schemas.observable import ObservableType
+
+class RegistryKey(Observable):
+    type: ObservableType = ObservableType.registry_key
+    key: str
+    value: str
+    hive: str | None = None
+    path_file: str | None = None
```

Comment:
What do you mean by this? 

I would also add a docstring to this class to explain what format things are expected to be in.